contracts-periphery/foundry.toml

```diff
@@ -1,5 +1,8 @@
 [default]
-remappings = ['test/=test/']
+remappings = ['test/=test/', '@uniswap/=lib/']
 [ci]
 fuzz_runs = 10000
+remappings = ['test/=test/', '@uniswap/=lib/']
+[hook]
+remappings = ['test/=test/', '@uniswap/=lib/']
```

Comment:
This profile seems like it's identical to the default one? Either way, as mentioned above I think this profile + the new remappings can be removed